ratpack-dropwizard-metrics/src/main/java/ratpack/dropwizard/metrics/DropwizardMetricsConfig.java

```diff
@@ -62,6 +64,48 @@ public DropwizardMetricsConfig jvmMetrics(boolean jvmMetrics) {
     return this;
   }
 
+  /**
+   * The state of byte buf allocator metrics collection.
+   *
+   * @return True if byte buf allocator metrics collection is enabled. False otherwise
+   * @since 1.6
+   */
+  public boolean isByteBufAllocatorMetrics() {
+    return byteBufAllocatorMetrics;
+  }
+
+  /**
+   * The state of request timing metrics reporting.
+   * @param byteBufAllocatorMetrics True if byte buf allocator metrics are to be reported. False otherwise
+   * @return this
+   * @since 1.6
+   */
+  public DropwizardMetricsConfig byteBufAllocatorMetrics(boolean byteBufAllocatorMetrics) {
+    this.byteBufAllocatorMetrics = byteBufAllocatorMetrics;
+    return this;
+  }
+
+  /**
+   * The state of detailed byte buf allocator metrics collection. (byte buf allocator metrics needs to be enabled)
+   *
+   * @return True if detailed byte buf allocator metrics collection is enabled. False otherwise
+   * @since 1.6
+   */
+  public boolean isDetailedByteBufAllocatorMetrics() {
```

Comment:
If there are multiple configuration options for this feature, then we should create a `ByteBufAllocatorConfig` object and nest the settings together rather than exposing both at the top level of `DropwizardMetricsConfig`.

See how `JmxConfig` is done in this class.